### DISCUSSION TITLE
Auto-reload not triggered upon making a change to HTML or adding Tailwind classes (unless AlpineJS script is commented out)

### DISCUSSION THREAD

**Discussion by nikoescobal** [alpinejs/alpine#3049] [7. Bugs (archived)]
# 🐛 bug report: 

Auto-reload is not triggered upon making a change to HTML or adding Tailwind classes (unless AlpineJS script is commented out)

Page doesn't auto-reload after applying changes to HTML or when adding Tailwind classes. I have to reload manually to get the changes to apply. However, when I comment out the CDN from AlpineJS, it works just fine.

## 🎛 Configuration (.babelrc, package.json, cli command)

Here's what my package.json config looks like:

```
{
  "name": "kickoff",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "parcel src/index.html --open",
    "build": "parcel build src/index.html",
    "test": "npm run eslint && echo \"✅✅✅ All tests passed ✅✅✅\"",
    "eslint": "eslint --fix"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/nikoescobal/kickoff.git"
  },
  "keywords": [],
  "author": "",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/nikoescobal/kickoff/issues"
  },
  "homepage": "https://github.com/nikoescobal/kickoff#readme",
  "devDependencies": {
    "@babel/eslint-parser": "^7.18.9",
    "@parcel/css": "^1.12.0",
    "@parcel/transformer-sass": "^2.6.2",
    "eslint": "^8.20.0",
    "eslint-config-airbnb-base": "^15.0.0",
    "eslint-plugin-import": "^2.26.0",
    "postcss-cli": "^10.0.0", 
    "tailwindcss": "^3.1.6",
    "parcel": "^2.6.2"
  },
  "dependencies": {
    "alpinejs": "^3.10.2"
  }
}

```

## 🤔 Expected Behavior

Making to the HTML or adding Tailwind classes should automatically trigger an auto-reload from Parcel.

## 😯 Current Behavior

I have to manually reload for the HTML changes/Tailwind classes to apply. However, if I uncomment the AlpineJS CDN, it works just fine.

## 💁 Possible Solution

Commenting out the AlpineJS script makes the file work, but then this defeats the purpose of being able to quickly build with both AlpineJS and TailwindCSS, given that I'll have to comment and then uncomment the CDN each time I'll be adding new methods, etc.

## 🔦 Context

I'm trying to set up a template so I can easily start a project with Parcel, TailwindCSS, and AlpineJS configured. 

## 💻 Code Sample

[Link to the repo](https://github.com/nikoescobal/kickoff)

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| TailwindCSS | ^3.1.6
| Parcel           | ^2.6.2
| Node            | v16.16.0
| npm/Yarn     | 8.5.5
| Alpine           | ^3.10.2
| Operating System | MacOS Monterey v12.4

**Comment by SimoTod** [comment]
I think you should open a bug on the parcel project if you want anyone to have a look at it, Alpine doesn't offer any autoreloading and even if it's caused by the interaction between Alpine and Parcel, the relevant code won't be in this repo.
